airbyte-integrations/connectors/source-file/source_file/spec.json

```diff
@@ -33,12 +33,13 @@
         "type": "string",
         "title": "Reader Options",
         "description": "This should be a string in JSON format. It depends on the chosen file format to provide additional options and tune its behavior.",
-        "examples": ["{}", "{\"sep\": \" \"}"]
+        "examples": ["{}", "{\"sep\": \" \"}", { "sep" : "\t", "header" : 0, "names": "column1, column2"}]
       },
       "url": {
         "type": "string",
         "title": "URL",
-        "description": "The URL path to access the file which should be replicated."
+        "description": "The URL path to access the file which should be replicated. The format for that is https://drive.google.com/uc?export=download&id=[DRIVE_FILE_ID] where [DRIVE_FILE_ID] is the string found in the Share URL here https://drive.google.com/file/d/[DRIVE_FILE_ID]/view?usp=sharing",
```

Comment:
I don't really get the example here. Isn't it a simple URL and not a portion of it as you mentioned in the example?

Reply:
thanks, I removed my new description, because it was for [Google Drive case](https://docs.airbyte.com/integrations/sources/file#getting-started), and added examples from [this table](https://docs.airbyte.com/integrations/sources/file#examples).